GitUI/CommandsDialogs/SettingsDialog/RepoDistSettingsSet.cs

```diff
@@ -2,23 +2,9 @@
 
 namespace GitUI.CommandsDialogs.SettingsDialog
 {
-    public class RepoDistSettingsSet
-    {
-        public readonly RepoDistSettings EffectiveSettings;
-        public readonly RepoDistSettings LocalSettings;
-        public readonly RepoDistSettings RepoDistSettings;
-        public readonly RepoDistSettings GlobalSettings;
-
-        public RepoDistSettingsSet(
-            RepoDistSettings effectiveSettings,
-            RepoDistSettings localSettings,
-            RepoDistSettings pulledSettings,
-            RepoDistSettings globalSettings)
-        {
-            EffectiveSettings = effectiveSettings;
-            LocalSettings = localSettings;
-            RepoDistSettings = pulledSettings;
-            GlobalSettings = globalSettings;
-        }
-    }
+    public readonly record struct RepoDistSettingsSet(
```

Comment:
Rename this struct too?
```suggestion
    public readonly record struct DistributedSettingsSet(
```